src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/ConfigTest.java

```diff
@@ -98,12 +101,37 @@ protected BaseCommandArguments createBaseCommandArguments() throws IOException {
         baseCommandArguments.setAppName("demoApp");
         baseCommandArguments.setBuildName("1");
         baseCommandArguments.setBranchName("branchy");
-        baseCommandArguments.setBuildSessionIdFile("/path/to/buildsessionid.txt");
         baseCommandArguments.setEnvVars(new EnvVars());
-
+        baseCommandArguments.setLogConfiguration(new LogConfiguration());
         TestHelper.BuildMock build = testHelper.createBuildMock();
         baseCommandArguments.setBuild(build);
 
         return baseCommandArguments;
     }
+
+    private List<String> createExpectedCommand(BaseCommandArguments baseCommandArguments) {
+        List<String> expected = new ArrayList<>();
+        expected.add("path/to/java");
+        expected.add(AbstractCommandExecutor.formatTagProp());
+        expected.addAll(baseCommandArguments.getLogConfiguration().toSystemProperties());
```

Comment:
Please test explicitly the field names.

Reply:
Here? this is test class for config command, the real values are tested in `LogConfiurationTest` (all the permutations)